package.json

```diff
@@ -49,6 +49,7 @@
     "test:integration": "jest --config ./jest.integration.js",
     "test:bundle:esm:browser": "cross-env BUNDLE_ENV=browser NODE_OPTIONS=--experimental-vm-modules jest --config ./jest.esm.mjs",
     "test:bundle:esm:node": "cross-env BUNDLE_ENV=node NODE_OPTIONS=--experimental-vm-modules jest --config ./jest.esm.mjs",
+    "test:bundle:esm:node:app": "yarn workspace @okta/test.app.node-esm start",
```

Comment:
is it possible to move this test to unit layer? catch error from a running node app seems like an integration test.

what if you don't map the module in jest config above?

Reply:
@shuowu 
> what if you don't map the module in jest config above?

Without module mapping in Jest `yarn test:bundle:esm:bowser` will fail with
`SyntaxError: Unexpected token 'export'`

It happens because ES module of `broadcast-channel` for browser (`dist/esbrowser/index.js`)  have '.js' extension (not '.mjs').

For the same reason Jest uses special ES module of `okta-auth-js` for browser with `.mjs` extension - `build/bundles-for-validation/esm/esm.browser.mjs` instead of `build/esm/esm.browser.js`. 
See 
https://github.com/okta/okta-auth-js/blob/7077d6369afdf567df5e0bdd367fb3b9d0774283/rollup.config.js#L93

Reply:
> is it possible to move this test to unit layer? catch error from a running node app seems like an integration test.

There is already unit test for validating bundles - `test:bundle:esm:node`

It has a little "hack" currently in master
https://github.com/okta/okta-auth-js/blob/7077d6369afdf567df5e0bdd367fb3b9d0774283/jest.esm.mjs#L32
to resolve `broadcast-chanel` with ES5 version of bundle which doesn't have `import/export`
I've replaced it with 
```js
'^broadcast-channel$': '<rootDir>/node_modules/broadcast-channel/dist/esnode/index.mjs' 
```

Reply:
```
const OktaAuth = process.env.BUNDLE_ENV === 'browser' ? 
  `<rootDir>/build/bundles-for-validation/esm/esm.browser.mjs` : 
  `<rootDir>/build/esm/esm.node.mjs`;
```

Is it possible to follow logic here to only apply module map for browser env, and leave it to jest to figure out the node dependencies. If possible, we should avoid over testing dependencies of auth-js.